tests/baselines/reference/textChanges/extractMethodLike.js

```diff
@@ -1,8 +1,8 @@
 ===ORIGINAL===
 
-namespace M 
```

Comment:
why did these change?

Reply:
I hit save in `unittests/textChanges.ts` and vscode automatically removed trailing whitespace from tests.
Moved that change to #19836.

Reply:
ah sorry.. i thought the code change somehow changed the baselines..